Cards/MadGraph5_aMCatNLO/SingleT/TWminusto2L2Nu-5FS-DR1_TuneCP5_13p6TeV_amcatnlo-pythia8/TWminusto2L2Nu-5FS-DR1_TuneCP5_13p6TeV_amcatnlo-pythia8_madspin_card.dat

```diff
@@ -0,0 +1,11 @@
+set ms_dir ./madspingrid
+
+set Nevents_for_max_weight 250
+set max_weight_ps_point 400
+
```

Comment:
set max_running_process 1 let's add this. this ensures that madspin runs on only 1 core (which is obviously not so efficient for gridpacks, but makes it thread safe during the production)